meinberlin/templates/base.html

```diff
@@ -23,8 +23,14 @@
     {% block extra_css %}
         {# Override this in templates to add extra stylesheets #}
     {% endblock %}
-    <link rel="stylesheet" type="text/css" href="{% static 'berlin_css.css' %}" />
-    <link rel="stylesheet" type="text/css" href="{% static 'adhocracy4.css' %}" />
+
+    <!-- FIXME this is to ensure the dashboard styles are untouched by BO styles but not sure its the best way? -->
+    <link rel="stylesheet" type="text/css" href="{% static 'common_css.css' %}" />
+    {% if '/dashboard' in request.path %}
+        <link rel="stylesheet" type="text/css" href="{% static 'adhocracy4.css' %}" />
+    {% else %}
+        <link rel="stylesheet" type="text/css" href="{% static 'berlin_css.css' %}" />
+    {% endif %}
```

Comment:
This is my first attempt to separate out the styles as the dashboard is getting affected, I don't think this is a perfect solution but to keep everything workable and to notice when deleted or altered styles will be problematic in the dashboard I think its a good idea but also happy for other suggestions?

Reply:
A more standard way is to work with blocks. E.g here you can make use of the above extra_css block and point to the berlin CSS, and in the meinberlin/templates/a4dashboard/base_dashboard.html you override it with the a4 CSS.